### PR TITLE
Fix SSL issue for upgrading

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**

### CODE REVIEW

File: protostar/commands/upgrade/upgrade_command.py [hunk 1]
```diff
@@ -89,9 +88,10 @@ def upgrade(self):
             self.current_version,
             self.latest_version,
         )
+
+        self._pull_tarball()
```
Comment (kasperski95) [inline protostar/commands/upgrade/upgrade_command.py:92]: Why did you move this call out of `try`'s scope?

**Review by kasperski95 — APPROVED** [review]
LGTM

File: protostar/commands/upgrade/upgrade_command.py (hunk 1 above)
Comment (MaksymilianDemitraszek) [reply protostar/commands/upgrade/upgrade_command.py:92]: Because i have to pull before backup, and there is nothing to cleanup if it fails.

Comment (MaksymilianDemitraszek) [reply protostar/commands/upgrade/upgrade_command.py:92]: backup moves cert location

Comment (MaksymilianDemitraszek) [reply protostar/commands/upgrade/upgrade_command.py:92]: good point 😅
